### PR TITLE
Add workflow for updating testfm image on quay

### PR DESCRIPTION
**Pull request by Gauravtalreja1**
Signed-off-by: Gaurav Talreja <gtalreja@redhat.com>

### CODE REVIEW

File: .github/workflows/merge_to_master.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.6'
+
+      - name: Install Dependencies
+        run: |
+          sudo apt update ; sudo apt-get install -y python-dev
+          pip install -r requirements.txt
+
+      - name: Install Dependencies
```
Comment (vsedmik) [inline .github/workflows/merge_to_master.yml:25]: Duplicity of the previous step.

File: .github/workflows/merge_to_master.yml [hunk 2]
```diff
@@ -0,0 +1,70 @@
+# CI stages to execute against master branch on PR merge
```
Comment (jameerpathan111) [inline .github/workflows/merge_to_master.yml:1]: This workflow will run against all branches not just master. We should update this comment and name of this yml file.

Comment (Gauravtalreja1) [reply .github/workflows/merge_to_master.yml]: @jameerpathan111 Updated.

**Review by jameerpathan111 — APPROVED** [review]
ACK

**Review by vsedmik — APPROVED** [review]
LGTM :+1:

**Pull request merged**